examples/nat/main.tf

```diff
@@ -14,13 +14,23 @@
  * limitations under the License.
  */
 
+module "vpc" {
+  source  = "terraform-google-modules/network/google"
+  version = "~> 7.0"
+
+  project_id   = var.project_id
+  network_name = "test-network"
+  routing_mode = "GLOBAL"
+  subnets      = []
+}
+
+
 # [START cloudnat_simple_create]
 module "cloud_router" {
-  source  = "terraform-google-modules/cloud-router/google"
```

Comment:
The above `# [START cloudnat_simple_create]` indicates this is part of a c.g.c doc. Could we keep the registry references? If this is used in a test, we can use module swapper as a build step to swap in local references.

https://github.com/terraform-google-modules/terraform-google-network/blob/6b1334d8b24c12082aa9d9f92e400cc1d16461e3/build/int.cloudbuild.yaml#L17